lib/sles4sap/ipaddr2.pm

```diff
@@ -48,6 +49,16 @@ our @EXPORT = qw(
   ipaddr2_test_other_vm
   ipaddr2_cloudinit_create
   ipaddr2_cloudinit_logs
+  ipaddr2_clean_network_peering
+  ipaddr2_azure_resource_group
+  ipaddr2_ssh_internal
+  ipaddr2_bastion_ssh_addr
+  ipaddr2_get_internal_vm_private_ip
+  ipaddr2_network_peering
+  ipaddr2_patch_system
+  ipaddr2_register_addons
+  ipaddr2_registration
```

Comment:
new function `ipaddr2_registration` has been created but there was already two functions in places to deal with registration
```
ipaddr2_registeration_check
ipaddr2_registeration_set
```